scripts/populate-instance.sh

```diff
@@ -106,7 +106,7 @@ ${INVENIO_WEB_INSTANCE} db create
 
 # sphinxdoc-create-user-account-begin
 ${INVENIO_WEB_INSTANCE} users create \
-       --email ${INVENIO_USER_EMAIL} \
+       ${INVENIO_USER_EMAIL} \
```

Comment:
Maybe pass first the optional arguments and second the mandatory arguments, as is the usual style?
(`invenio3 users create --foo1 bar1 --foo2 bar2 xyzzy zyxxy`)